src/plugins/telemetryTable/TelemetryTableConfiguration.js

```diff
@@ -99,16 +99,18 @@ define([
         }
 
         hasColumnWithKey(columnKey) {
-            return _.flatten(Object.values(this.columns))
-                .findIndex(column => column.getKey() === columnKey) !== -1;
+            const flatten = arr => arr.reduce((a, b) => a.concat(b), []);
+            return flatten(Object.values(this.columns))
+                .some(column => column.getKey() === columnKey);
         }
 
         getColumns() {
             return this.columns;
         }
 
         getAllHeaders() {
-            let flattenedColumns = _.flatten(Object.values(this.columns));
+            const flatten = arr => arr.reduce((a, b) => a.concat(b), []);
+            let flattenedColumns = flatten(Object.values(this.columns));
```

Comment:
The name of this function should describe the level at which arrays will be flattened. This seems to be one level deep only right?

Reply:
Will fix this

Reply:
How about this solution using recursion?
```
var arr1 = [1,2,3,[1,2,3,4, [2,3,4]]];
function flattenDeep(arr1) {
   return arr1.reduce((acc, val) => Array.isArray(val) ? acc.concat(flattenDeep(val)) : acc.concat(val), []);
}
flattenDeep(arr1);// [1, 2, 3, 1, 2, 3, 4, 2, 3, 4]

```

Reply:
added this as another option in array-utils